prompts.js

```diff
@@ -55,4 +48,4 @@ const prompts = [
   }
 ]
 
-module.exports = !hasCordova ? cordovaBinaryMissing : prompts
+module.exports = !hasCordova ? null : prompts
```

Comment:
Before, it was ok, to negate hasCordova, because, you did something then... Now, you do nothing, so IMO it's more intuitive and understandable if you do: 

```javascript
module.exports = hasCordova ? prompts : null
```